scripts/actions/flows.actions.js

```diff
@@ -34,6 +34,14 @@ export function selectColumn(columnIndex, columnId) {
   };
 }
 
+export function selectVectorLayers(vectorLayers, reloadLinks) {
+  return _getSelectAction('vectorLayers', vectorLayers, actions.SELECT_VECTOR_LAYERS, reloadLinks);
+}
+
+export function selectContextualLayers(contextualLayers, reloadLinks) {
+  return _getSelectAction('contextualLayers', contextualLayers, actions.SELECT_CONTEXTUAL_LAYERS, reloadLinks);
```

Comment:
`_getSelectAction` was a wrong choice of name, can you rename it `_reloadLinks` or something like this plz?